doc/man1/s_server.pod

```diff
@@ -245,6 +245,17 @@ certificate and some require a certificate with a certain public key type:
 for example the DSS cipher suites require a certificate containing a DSS
 (DSA) key. If not specified then the filename "server.pem" will be used.
 
+=item B<-cert_chain>
+
+A file containing trusted certificates to use when attempting to build the
```

Comment:
when .. when
in the same sentence .

Reply:
...to use during building the server certificate chain?

Reply:
`A file containing trusted certificates to use when attempting to build the client/server certificate chain related to the certificate specified via the B<-cert> option .`
-> there is the same construct in this client and server side of doc.

Reply:
Thank you, the suggested rephrasing is applied.